fluent-react/src/localized.ts

```diff
@@ -1,4 +1,5 @@
-import {
+import React, {
+  isValidElement,
```

Comment:
```suggestion
import React, {
  Children,
  isValidElement,
```

Reply:
This was part of https://github.com/projectfluent/fluent.js/pull/595#discussion_r1006189103.